project_name/__init__.py

```diff
@@ -1,2 +1,4 @@
 # flake8: noqa
 from .version import __version__
+
+django_app_config = '{{ project_name | replace("_", " ") | title | replace(" ", "") }}Config'
```

Comment:
This should be `django_app_confg = 'project_name.apps.{{ project_name | replace("_", " ") | title | replace(" ", "") }}Config'`